engine/swarm/key-concepts.md

```diff
@@ -14,30 +14,33 @@ are built using **SwarmKit**. Engines participating in a cluster are
 running in **swarm mode**. You enable swarm mode for the Engine by either
 initializing a swarm or joining an existing swarm.
 
-A **swarm** is a cluster of Docker Engines where you deploy
-[services](key-concepts.md#Services-and-tasks). The Docker Engine CLI includes the commands for
-swarm management, such as adding and removing nodes. The CLI also includes the
-commands you need to deploy services to the swarm and manage service
-orchestration.
+A **swarm** is a cluster of Docker engines, or _nodes_, where you deploy
+[services](key-concepts.md#Services-and-tasks). The Docker Engine CLI includes
+the commands for swarm management, such as adding and removing nodes. The CLI
+also includes the commands you need to deploy services to the swarm and manage
+service orchestration.
 
-When you run Docker Engine outside of swarm mode, you execute container
-commands. When you run the Engine in swarm mode, you orchestrate services.
+When you run Docker outside of swarm mode, you execute container
```

Comment:
maybe instead of "outside of swarm mode", say something like "without using a swarm" or something? Also this kind of implies you can't run services and containers in the same Engine.